docs/developers/contributing.md

```diff
@@ -9,55 +9,95 @@ If you are a new user, we recommend checking out the detailed [Github Docs](http
-`skip-string-normalization = true` and `max-line-length = 79`. To learn more,
-see [`black`'s documentation](https://black.readthedocs.io/en/stable/).
-
-Code will also be linted to enforce the stylistic and logistical rules specified
-in our [`flake8` configuration](https://github.com/napari/napari/blob/main/setup.cfg), which currently ignores
-[E203](https://lintlyci.github.io/Flake8Rules/rules/E203.html),
-[E501](https://lintlyci.github.io/Flake8Rules/rules/E501.html),
-[W503](https://lintlyci.github.io/Flake8Rules/rules/W503.html) and
-[C901](https://lintlyci.github.io/Flake8Rules/rules/C901.html).  For information
-on any specific flake8 error code, see the [Flake8
-Rules](https://lintlyci.github.io/Flake8Rules/).  You may also wish to refer to
-the [PEP 8 style guide](https://peps.python.org/pep-0008/).
-
-If you wish to tell the linter to ignore a specific line use the `# noqa`
-comment along with the specific error code (e.g. `import sys  # noqa: E402`) but
-please do not ignore errors lightly.
+[repository](https://github.com/napari/napari). If you are not familiar with `git`, we recommend reading up on [this guide](https://docs.github.com/en/get-started/using-git/about-git#basic-git-commands).
+
+1. Clone the forked repository to your local machine and change directories:
+
+    ```sh
+    git clone https://github.com/your-username/napari.git
+    cd napari
+    ```
+
+2. Set the `upstream` remote to the base `napari` repository:
+
+    ```sh
+    git remote add upstream https://github.com/napari/napari.git
+    ```
+
+3. If you haven't already, create a development environment:
+
+    ::::{tab-set}
+
+    :::{tab-item} Using `venv`
+    After installing Python on your machine, create a virtual environment on your terminal and activate it. On Linux and MacOS, you can run
+    ```sh
+    python -m venv <path-to-env>
+    source <path-to-env>/bin/activate
+    ```
+    See the [venv](https://docs.python.org/3/library/venv.html) documentation for instructions on Windows.
+    :::
+
+    :::{tab-item} Using `conda`
+    After [installing `conda`](https://www.anaconda.com/products/distribution), create an environment called `napari-env` with Python 3.9 and activate it.
+    ```sh
+    conda create -y --name napari-env -c conda-forge python=3.9
+    conda activate napari-env
+    ```
+    :::
+
+    ::::
+
+    ```{note}
+    It is highly recommended to create a fresh environment when working with
+    napari, to prevent issues with outdated or conflicting packages in your
+    development environment.
+    ```
+
+4. Install the package in editable mode, along with all of the developer tools.
+
+    If you only want to use napari, you can install it on most macOS, Linux and
+    Windows systems with
+    ![Supported Python versions](https://img.shields.io/pypi/pyversions/napari.svg)
+    by following the directions on the
+    [instructions page](installation#install-as-python-package-recommended).
```

Comment:
```suggestion
    [instructions page](../tutorials/fundamentals/installation.md#install-as-python-package-recommended).
```
For me the current link is broken, I think this should be correct

Reply:
Not sure I wouldn't just remove this paragraph about the standard installation anyways.